mne/io/cnt/cnt.py

```diff
@@ -60,6 +61,12 @@ def read_raw_cnt(input_fname, montage, eog=(), misc=(), ecg=(), emg=(),
         Names of channels or list of indices that should be designated
         EMG channels. If 'auto', the channel names containing 'EMG' are used.
         Defaults to empty tuple.
+    n_bytes : int
+        Defines the number of bytes the data is read in (should be 2 for int16
+        and 4 for int32). Defaults to 2.
```

Comment:
I still don't have matlab, so I'm not able to make one (I think it's possible to write cnt files with EEGLAB) and I don't have any data in int32.